Lib/pathlib/__init__.py

```diff
@@ -261,6 +261,16 @@ def is_relative_to(self, other, /, *_deprecated):
             other = self.with_segments(other, *_deprecated)
         return _abc.PurePathBase.is_relative_to(self, other)
 
+    def is_reserved(self):
+        """Return True if the path contains one of the special names reserved
+        by the system, if any."""
+        msg = ("pathlib.PurePath.is_reserved() is deprecated and "
+               "scheduled for removal in Python 3.15")
```

Comment:
Would be great to reference `ntpath.isreserved` from here for Windows users, and probably to mention that other platforms don't have reserved names (if that's true... I'm not sure exactly what wording makes sense for that)